desktop/apps/auctions2/templates/index.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { withLayout } from 'desktop/components/react/main_layout'
+
+function Index (props) {
+  const {
+    Layout: { Header, Body, Footer },
+    Templates: { Meta }
+  } = props
+
+  return (
+    <div>
+      <Header>
+        <Meta />
+      </Header>
+      <Body>
+        hello!
+      </Body>
+      <Footer />
+    </div>
```

Comment:
Like, it seems to me that a given app is either going to use the old `block` way to insert content into its subsections or it will want to manage that content using a React component. That's where I'm coming from here -- if it's the old way, then why have to mention these `Layout` subsections at all? If it's the new way, then rather than trying to control those subsections in one super component that is responsible for composing the layout with the main content and custom subsections, does it make more sense to just provide the main content and custom subsections as separate pieces of data, to be composed into the layout by one central piece of functionality. 

So basically, I'm suggesting separating the concerns of full-page layout composition and app specific content generation by pulling those layout concerns out of the app-specific code, and just having the app-specific code supply the layout data, not the layout logic.